.colab/00_lineapy_quickstart.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
we seem to have two notebooks starting with 00_?

Reply:
Intention is to have `00_lineapy_quickstart.ipynb` and `01_using_artifacts.ipynb` replace existing `00_api_basics.ipynb`.

We are retaining `00_api_basics.ipynb` in `.colab/` temporarily because we need existing links to work (which can be updated once this PR gets merged into `main` and `main` into `v0.2.x`). Once we route users to the new notebooks, `00_api_basics.ipynb` will be dropped.

Reply:
As per discussion with Shardul, I am removing `00_api_basics.ipynb` from `.colab/`.